README.rst

```diff
@@ -86,9 +85,11 @@ You will need to implement the following methods:
             """
             return "This thing is really cool, you should totally check it out"
 
-        def get_meta_image(self):
-            """A relevant Wagtail Image to show. Optional."""
-            return self.some_image
+        def get_meta_image_url(self):
+            """
+            Return a url for an image to use, see the MetadataPageMixin if using a Wagtail image
+            """
+            return 'https://neonjungle.studio/share'
```

Comment:
Should this URL have .jpg/.png on it to help make it clear to users that it should return an image of some kind?